Source/Core/Core/HW/DSPHLE/UCodes/ASnd.h

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <utility>
+
+#include "Common/CommonTypes.h"
+#include "Core/HW/DSPHLE/UCodes/UCodes.h"
+
+namespace DSP::HLE
+{
+class DSPHLE;
+
+struct ASndUCode : public UCodeInterface
```

Comment:
Parent is a `class`, I recall visual studio complaining about this.  Seeing as most everything is private anyway, maybe make it a `class` too?

Also `final`

Reply:
It looks like I based this on `GBAUCode` which is also a `struct` for some reason.  I'll change `GBAUCode` in a separate PR.

Reply:
#10768